MIP0/mip0.md

```diff
@@ -7,287 +11,254 @@ Title: The Maker Improvement Proposal Framework
 Author(s): Charles St.Louis (@CPSTL), Rune Christensen (@Rune23)
 Contributors: @LongForWisdom
 Type: Process
-Status: Accepted 
+Status: Accepted
 Date Proposed: 2020-04-06
 Date Ratified: 2020-05-02
 Dependencies: n/a
 Replaces: n/a
 ```
 ## References
 **[General-MIP-Template.md](General-MIP-Template.md)**  
-**[Technical-MIP-Template.md](Technical-MIP-Template.md)**  
-**[MIP0c12-Subproposal-Template.md](MIP0c12-Subproposal-Template.md)**  
-**[MIP0c13-Subproposal-Template.md](MIP0c13-Subproposal-Template.md)**  
+**[Technical-MIP-Template.md](Technical-MIP-Template.md)**
 
 ## Sentence Summary
 
-MIP0 defines and describes the MIPs Framework. 
+MIP0 defines the *Maker Improvement Proposals (MIPs) Framework* for all subsequent MIPs to utilize.
 
 ## Paragraph Summary
 
-MIP0 is the genesis proposal describing the MIPs Framework. This includes the core components and statuses as well as the various MIP types and the overall MIP lifecycle. Furthermore, it provides the necessary tools, such as MIP templates, replacement processes, and dependencies. Lastly, the proposal details the key roles of the framework, the MIP Editor and the Governance Facilitator along with the process for adding and removing them.
+MIPs are the preferred mechanism for improving Maker Governance and the Maker Protocol. MIP0 defines the *Maker Improvement Proposals Framework* for all subsequent MIPs to utilize.
```

Comment:
I would start this off with the sentence summary so: "MIP0 defines the *Maker Improvement Proposals (MIPs) Framework* for all subsequent MIPs to utilize." Then whatever...